tests/publiccloud/boottime.pm

```diff
@@ -423,7 +423,7 @@ sub run {
 
     my $results = $self->measure_timings($args);
     $self->store_in_db($results) if (check_var('_PUBLIC_CLOUD_PERF_PUSH_DATA', 1));
-    $self->check_thresholds($results);
+    $self->check_thresholds($results) if (check_var('PUBLIC_CLOUD_PERF_THRESH_CHECK', 1));
```

Comment:
```suggestion
    $self->check_thresholds($results) if check_var('PUBLIC_CLOUD_PERF_THRESH_CHECK');
```

I believe any value that translates to True in Perl is enough.

Reply:
no check_var won't work this way : 

```
sub check_var {
    my ($var, $val) = @_;
    return 1 if (defined $bmwqemu::vars{$var} && $bmwqemu::vars{$var} eq $val);
    return 0;
}
```

Reply:
so if $val will be undef it will always return 0

Reply:
In this case I choosed the only value ` 1 ` as explicit intention to enable the checks. But since I named it in variables.md to be boolean, indeed any value must be ok.

Reply:
Sorry, I meant `get_var()`